### PR TITLE
ci: Remove the body length requirement

### PR DESCRIPTION
**Pull request by thewilloftheshadow**

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ module.exports = {
     extends: ['@commitlint/config-conventional'],
     rules: {
         'subject-case': [1, 'always', ['lower-case']],
-        'body-max-line-length': [0],
+        'footer-leading-blank': 'never'
```
Comment (not-so-smart) [inline commitlint.config.js:5]: wait why did you remove the body max length rule?
also shouldn't we add a rule for footer max length too?

Comment (thewilloftheshadow) [reply commitlint.config.js:5]: The docs says the default for both is infinity
